docker/layotto_injector/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.18-alpine AS build-env
+
+WORKDIR /app
+
+COPY . .
+
+RUN go build -o injector cmd/layotto_injector/main.go
```

Comment:
`make image IMAGES=layotto_injector`

Make sure this command works well. Refer to `docker/layotto/dockerfile`.